build.gradle

```diff
@@ -91,58 +90,60 @@ publishing {
 }
 
 def versions = [
-    spring : '2.3.5.RELEASE',
+    spring : '2.6.3',
     lombok: '1.18.22'
 ]
 
 dependencies {
     api("com.google.guava:guava:31.0.1-jre")
     api("org.apache.commons:commons-lang3:3.12.0")
-    compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.12.5'
-    compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jdk8', version: '2.12.5'
-    compile group: 'com.google.code.gson', name: 'gson', version: '2.8.9'
-    compile group: 'com.googlecode.json-simple', name: 'json-simple', version: '1.1.1'
-    compile group: 'com.github.hmcts', name: 'java-logging', version: '5.1.9'
-    compile group: 'com.github.hmcts', name: 'ccd-client', version: '4.8.4'
-    compile group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '9.15.2'
 
-    compile group: 'io.springfox', name: 'springfox-swagger2', version: '3.0.0'
-    compile group: 'com.github.everit-org.json-schema', name: 'org.everit.json.schema', version: '1.13.0'
-    compile group: 'org.elasticsearch', name: 'elasticsearch', version: '7.15.1'
+    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.12.5'
+    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jdk8', version: '2.12.5'
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.9'
+    implementation group: 'com.googlecode.json-simple', name: 'json-simple', version: '1.1.1'
+    implementation group: 'com.github.hmcts', name: 'java-logging', version: '5.1.9'
+    implementation group: 'com.github.hmcts', name: 'ccd-client', version: '4.8.6'
+    implementation group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '9.15.2'
 
-    compile group: 'org.springframework.retry', name: 'spring-retry', version: '1.3.1'
-    compile group: 'io.github.openfeign.form', name: 'feign-form', version: '3.8.0'
-    compile group: 'com.github.hmcts', name: 'service-auth-provider-java-client', version: '4.0.2'
+    implementation group: 'io.springfox', name: 'springfox-swagger2', version: '3.0.0'
```

Comment:
In my opinion it's cleaner to move the versions into the versions block above so they're in the same place

Reply:
PMD ought to be bumped too

Reply:
I think we only move the versions for the ones that are most commonly use, I can update it if you prefer? Yeah I can bump PMD as well!